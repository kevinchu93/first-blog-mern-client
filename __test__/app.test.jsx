import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import App from '../src/app';
import Header from '../src/app/header';

let wrapper;

describe('App Component - shallow', () => {
  beforeEach(() => {
    wrapper = shallow(<App />);
  });
  test('App component should render as expected - snapshot', () => {
    expect(toJson(wrapper)).toMatchSnapshot();
  });
  test('should render Header component', () => {
    expect(wrapper.containsMatchingElement(<Header />)).toEqual(true);
  });
});
