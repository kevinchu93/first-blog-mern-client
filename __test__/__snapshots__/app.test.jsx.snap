// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Component - shallow App component should render as expected - snapshot 1`] = `
<div>
  <Header />
  <div
    className="body"
  >
    <Route
      exact={true}
      path="/"
      render={[Function]}
    />
    <Route
      exact={true}
      path="/posts"
      render={[Function]}
    />
    <Route
      component={[Function]}
      path="/posts/post_id"
    />
    <Route
      component={[Function]}
      path="/post_new"
    />
  </div>
</div>
`;
